mender-update/daemon/state_test.cpp

```diff
@@ -2693,6 +2831,15 @@ class StateDeathTest : public testing::TestWithParam<StateTransitionsTestCase> {
 	mtesting::TemporaryDirectory tmpdir_;
 };
 
+INSTANTIATE_TEST_SUITE_P(
+	Regular_Non_Update_State_Tests,
+	StateDeathTest,
+	::testing::ValuesIn(idle_and_sync_test_cases),
```

Comment:
Why is this a separate data set?

Reply:
I figured cause it's a separate part of the state-machine, and it makes running them separately easier, but can merge them if you prefer.

Reply:
No, it's fine. I was just a bit confused since you added this, but didn't add a test, so I just wondered if you had forgotten something.